metrics-operator/controllers/common/providers/dynatrace/client/client.go

```diff
@@ -89,10 +89,10 @@ func (client *apiClient) Do(ctx context.Context, path, method string, payload []
 	return b, res.StatusCode, nil
 }
 
-func (client *apiClient) auth(ctx context.Context) error {
+func (client *apiClient) auth(ctx context.Context) (int, error) {
```

Comment:
why is this change actually needed when we are ignoring any code that the function returns [here](https://github.com/keptn/lifecycle-toolkit/pull/2885/files#diff-1c9f73b5a8ee2d9c3f3c9b11a01fdaff1e2ba16e48704c404880cbf670a2606cR60)

This change does not seem to have any impact on the behavior of the operator. Or am I missing somethig?

Thank you!

Reply:
No. You are not missing anything 😃

My bad. I will return the status code from the auth instead of returning `http.StatusUnauthorized`